sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Improved FPGA archive device unbundling with AOCO. [4ed3676f6402]
+- Moved imf `abs` to a separate device library for Deep Learning. [e4f074a6aabe]
+- Fixed the bug report URL for DPC++. [98b7de88a7d9]
+
+### SYCL Library
+- Combined ADL-S and RPL-S device architectures. [f87be6f17428]
+- Implemented multi_ptr default to be legacy to avoid code break with SYCL 1.2.1. [52edb3798749]
+- Started to use aggregate initialization for `group_local_memory` arrays according to updated [sycl_ext_oneapi_local_memory](sycl/doc/extensions/supported/sycl_ext_oneapi_local_memory.asciidoc).[b50440ec76b9]
+- Deprecated ESIMD Emulator.  [177680e3e918]
+- Deprecated `ext::oneapi::sub_group`. [0662e2a81014]
+- Improved error message related to `ext_intel_free_memory` aspect. [c1bfcaf6d6ed]
+- Removed non-necessary barrier after region copy in the Level Zero backend. [768e61e7966c]
+- Fixed `get_info<device::free_memory>` to check its aspect. [dfb75675bc0a]
+- Removed the workaround for release of auxiliary buffers. [fda9171fc218]
+- Enabled optimization for read-only buffer accesses from multiple devices in a context. [dddaf7f5d54d]
+- Removed old specialization constants extension and implementation. [e50ae05e28d3][833a9fe0a9cf]
+- Added README.md for the Unified Runtime directory. [453ffdcf41dd]
+- Improved `is_compatible` to check if specific target is defined with `-fsycl-targets` and change the result. [ef033238cfb5]
+- Improved quotient approximation for host `sycl::remquo`. [d335fdf4afa2]
+- Made `bit_cast.hpp` self-contained. [c899a93410c2]
+- Allowed accessor constructed with zero-size buffers. [0c0809590f37]
+- Improved resources recycling for the Level Zero backend by cleaning up events on in-order queue wait. [260182a1ad75]
+- Added `code_location` parameter to the rest of `sycl::queue` methods which allows to improve error reporting. [fdd609a5c724]
+- Enabled `xpti::node_create` signal emit for parallel_for that bypasses graph. [4eaaaa963ca2]
+- Improved the accuracy of host `sycl::cospi`. [37aa84b7afd5]
+- Promoted the return type changes of SYCL relational builtins that changed between SYCL 1.2.1 and SYCL 2020 out from the guard of `SYCL2020_CONFORMANT_APIS`. [35fe4a21e1cb]
+- Replaced usage of outdated memset OpenCL API `clEnqueueMemsetINTEL` with `clEnqueueMemFillINTEL`. [c5d0e1d4d9be]
+- Added memory pointer to XPTI memory allocation event metadata to allow more informative XPTI memory tracing. [01ffe65da562]
+- Implemented recycling of immediate command lists for queues in a context for the Level Zero backend. [2ddbf8c3b7a5]
+- Optimized `ext_oneapi_submit_barrier()` for in-order queue for the Level Zero backend. [c21fd476c78a]
+- Removed `sycl_ext_oneapi_extended_atomics` extension. [17ea6d10f1d5]
+- Removed `sycl_ext_oneapi_group_algorithms` extension. [8b9d2106af0d]
+- Removed unneeded backwards compatibility of 2023.2 make_queue and get_native. [c5f150a41b1d][5918e0c208a1]
```

Comment:
```suggestion
- Removed unneeded backwards compatibility of 2023.2 `make_queue` and `get_native`. [c5f150a41b1d][5918e0c208a1]
```

Feels weird to see mentions of oneAPI versioning in the open-source repo where releases are named/versioned differently

Reply:
Fixed and removed confusing version.